templates/settings-admin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @copyright Copyright (c) 2016 Bjoern Schiessle <bjoern@schiessle.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/** @var array $_ */
+/** @var OC_L10N $l */
+script('password_policy', 'settings-admin');
+style('password_policy', 'settings-admin');
+?>
+
+<div id="password-policy" class="section">
+	<h2 class="inlineblock"><?php p($l->t('Password Policy')); ?></h2>
+	<div id="password-policy-settings-msg" class="msg success inlineblock" style="display: none;">Saved</div>
```

Comment:
That's just a placeholder and never displayed. The real string comes from here: https://github.com/nextcloud/password_policy/pull/1/files#diff-1376a269d6d13887451e97e2732f4dd4R30